### PR TITLE
Update CREATE FULLTEXT syntax summary to follow the rules we set up for such summaries

### PR DESCRIPTION
**Pull request by Hunterness**

### CODE REVIEW

**Comment by Hunterness** [comment]
[CREATE FULLTEXT INDEX includes | which is also used in syntax summaries for OR, should be escaped](https://trello.com/c/I8cYmdax/2699-create-fulltext-index-includes-which-is-also-used-in-syntax-summaries-for-or-should-be-escaped)

**Review by HannesSandberg — APPROVED** [review]
lgtm 👍

File: cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-full-text-search/create-fulltext-syntax.asciidoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 | [source, cypher, role=noplay]
 ----
 CREATE FULLTEXT INDEX [index_name] [IF NOT EXISTS]
-FOR (n:LabelName[\|...])
+FOR (n:LabelName["\|" ...])
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-full-text-search/create-fulltext-syntax.asciidoc:9]: Why is it `index_name` here but just `name` above? Also, why only one pipe here when two pipes in the example above?

**Review by AlexicaWright — COMMENTED** [review]
A few questions.

**Review by AlexicaWright — APPROVED** [review]

**Pull request merged**